COPYING

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
I just copied the licensing header from the existing files. If we want to relicense, that’s fine, but let’s do it as a separate commit to reduce the risk of confusion.